README.rst

```diff
@@ -65,15 +65,13 @@ Note that subsequent calls to ``pip install -e .`` will use cached binaries stor
 The following dependencies are required to install PennyLane-Lightning:
 
 * A C++ compiler, such as ``g++``, ``clang``, or ``MSVC``.
```

Comment:
Should we mention we want version 2014 and above now?

Reply:
So far the C++14 features that are used by the new backend could be replaced by a custom function. So we could stay with C++11 for now (unless another need arises).